src/lib.rs

```diff
@@ -141,6 +141,17 @@ where
         /// Amount of frames recommended to be skipped in order to let other clients catch up.
         skip_frames: u32,
     },
+    /// Sent whenever GGRS locally detected a discrepancy between local and remote checksums
+    DesyncDetected{
+        /// Frame of the checksums
+        frame: Frame,
+        /// local checksum for the given frame
+        local_checksum: u128,
+        /// remote checksum for the given frame
+        remote_checksum: u128,
+        /// remote address of the endpoint.
+        remote_addr: T::Address,
```

Comment:
Should perhaps just be `addr`, to match the other events. (For consistency)

Reply:
From testing with with my own game i found it quite nice to have the information available. i'd like to hear more opinions about it tho

Reply:
I just think either all the enum's addresses should be called addr, or all should be called remote_addr. There's no reason for the naming of this one to be special?

Reply:
Oh i misunderstood yeah good suggestion!

Reply:
yeah, I get how it's confusing now that I reread it :P